From hackermail Thu Jan  1 00:00:00 1970
From: SeongJae Park <sj@kernel.org>
To: Andrew Morton <akpm@linux-foundation.org>
Cc: SeongJae Park <sj@kernel.org>, Jonathan Corbet <corbet@lwn.net>, damon@lists.linux.dev, kernel-team@meta.com, linux-doc@vger.kernel.org, linux-kernel@vger.kernel.org, linux-mm@kvack.org
Message-Id: <20250218223708.53437-3-sj@kernel.org>
In-Reply-To: <20250218223708.53437-1-sj@kernel.org>
Date: Tue, 18 Feb 2025 14:37:05 -0800
Subject: [PATCH 2/5] Docs/mm/damon/design: document hugepage_size filter

'hugepage_size' DAMOS filter type is not documented on the design doc.
Add a description of the type.

Signed-off-by: SeongJae Park <sj@kernel.org>
Link: https://patch.msgid.link/20250218223708.53437-3-sj@kernel.org
---
 Documentation/mm/damon/design.rst | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/mm/damon/design.rst b/Documentation/mm/damon/design.rst
index 12ae7e1209c8..a959c081bc59 100644
--- a/Documentation/mm/damon/design.rst
+++ b/Documentation/mm/damon/design.rst
@@ -610,6 +610,9 @@ Below ``type`` of filters are currently supported.
     - Applied to pages that are accessed after the last access check from the
       scheme.
     - Handled by operations set layer.  Supported by only ``paddr`` set.
+- pages that managed in a given size range
+    - Applied to pages that managed in a given size range.
+    - Handled by operations set layer.  Supported by only ``paddr`` set.
 - address range
     - Applied to pages that belonging to a given address range.
     - Handled by the core logic.
-- 
2.39.5